client/README.md

```diff
@@ -20,7 +20,7 @@ It uses the HTTP protocol to communicate with your **InfluxDB** cluster.
 Connecting to an **InfluxDB** database is straightforward. You will need a host
 name, a port and the cluster user credentials if applicable. The default port is
 8086. You can customize these settings to your specific installation via the
-**InfluxDB** configuration file.
+[**InfluxDB** configuration file](https://influxdb.com/docs/v0.9/introduction/installation.html#generate-a-configuration-file).
```

Comment:
The links to docs has changed recently to https://docs.influxdata.com/influxdb/v0.10/.